src/screens/Home/components/ChainListItem.js

```diff
@@ -39,23 +38,19 @@ export type Props = {|
 |};
 
 function CategoryListItem({ title, onPress, value, isDeployed, onPressDeploy }: Props) {
-  const { t } = useTranslation();
   const colors = useThemeColors();
 
   return (
     <Container onPress={onPress}>
-      <Title>{title}</Title>
-
-      {isDeployed && !!value && (
-        <Value>{value}</Value>
-      )}
-
-      {!isDeployed && (
-        <DeployContainer onPress={onPressDeploy}>
-          <DeployValue>{t('button.deploy')}</DeployValue>
-          <Icon name="question" width={14} height={14} color={colors.labelTertiary} />
+      {!isDeployed ? (
+        <DeployContainer>
+          <TitleWithIcon onPress={onPress}>{title}</TitleWithIcon>
+          <HazardIcon name="warning" width={16} height={16} color={colors.hazardIconColor} onPress={onPressDeploy} />
```

Comment:
As the icon is small you probably need to add thing like `hitSlop={hitSlop20}` for user to be able to trigger the action.

Reply:
Done.